internal/cluster/query.go

```diff
@@ -0,0 +1,46 @@
+package cluster
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/buildkite/cli/v3/internal/graphql"
+	"github.com/buildkite/cli/v3/pkg/cmd/factory"
+)
+
+func QueryCluster(ctx context.Context, OrganizationSlug string, ClusterID string, f *factory.Factory) (Cluster, error) {
+
+	q, err := graphql.GetClusterQueues(ctx, f.GraphQLClient, OrganizationSlug, ClusterID)
+
+	if err != nil {
+		fmt.Println("Unable to read Cluster Queues: ", err.Error())
+		return Cluster{}, err
```

Comment:
Can we return the error with the error message using go's `fmt.Errorf` function, and also use the practice to return `nil` instead of an empty Cluster object?

Reply:
Done